docs/pubsub_snippets.py

```diff
@@ -311,24 +311,21 @@ def subscription_pull(client, to_delete):
     subscription.create()
     to_delete.append(subscription)
 
-    # [START subscription_pull_none_pending]
-    pulled = subscription.pull(max_messages=1)
-    # [END subscription_pull_none_pending]
```

Comment:
I just did a `git grep subscription_pull_none_pending` to make sure this snippet wasn't used anywhere. Why the `START` / `END` if it wasn't used?